circle/docker-release-image.sh

```diff
@@ -106,6 +106,13 @@ if [[ -n $GCLOUD_PROJECT && -n $GCLOUD_SERVICE_KEY ]]; then
   done
 fi
 
+if [[ -n $AZURE_PROJECT && -n $AZURE_PORTAL_PASS ]]; then
```

Comment:
Just in case you don't know:

`$AZURE_PROJECT`: This env should be defined in the circle.yml file for each application (Eg: https://github.com/bitnami/bitnami-docker-ghost/blob/master/circle.yml )
`$AZURE_PORTAL_PASS`: This env should be defined in the circle ci project configuration for each application